docs/schemes/local.md

```diff
@@ -32,6 +33,12 @@ Each endpoint is used to make requests using axios. They are basically extending
 
 > **TIP:** To disable each endpoint, simply set it's value to `false`.
 
+### `userinfo_decode`
```

Comment:
I think it would be better to be something like `jwt_decode` or something like that :)

Reply:
@pi0 @sschadwick Totally agree. And if #188 can be merge, 'jwt-decode' just will be part on oauth2.js.